PokeAlarm/Utils.py

```diff
@@ -429,7 +429,7 @@ def get_evolution_cost_chain(j, id_, form_id_, a=None):
         if j[id_].get("forms").get(form_id_).get("evolutions") is None:
             pass
         else:
-            for evo_id in j[id_].get("forms").get(form_id_).get("evolutions"):
+            for evo_id in j[id_].get("forms", {}).get(form_id_).get("evolutions", {}):
```

Comment:
I think this is still buggy if `"forms"` is {}.
In that case you `{}.get(form_id_)` which is `None` and then in turn `None.get("evolutions", {})` is evaluated.
Will fix in #865, so no need for action here.